imagefactory_plugins/ovfcommon/ovfcommon.py

```diff
@@ -1247,7 +1247,7 @@ def sync(self):
 
         vagrantfile = '''Vagrant.configure("2") do |config|
   config.vm.base_mac = "%s"
-  config.vm.synced_folder ".", "%s", type: "rsync"
+  config.vm.synced_folder ".", "/vagrant", disabled: true
```

Comment:
@imcleod I should have removed the line ` config.vm.synced_folder ".", "%s", type: "rsync"` instead of disabling it.